02LessonTwo/homework/index.html

```diff
@@ -1,10 +1,45 @@
 <!DOCTYPE html>
 <html>
 <head>
-  <title></title>
+  <title>Salvador Rodriguez Artwork</title>
   <link rel="stylesheet" href="css/style.css">
 </head>
+
 <body>
+  <!-- Defines page header -->
+  <div>
+    <img src="C:\Users\salva\OneDrive\Documents\GitHub\intro-workbook\02LessonTwo\homework\artwork1.jpg"/>
```

Comment:
Thanks Victoria. 😀
On May 10, 2016 9:58 AM, "Victoria Chuang" <notifications@github.com> wrote:

> In 02LessonTwo/homework/index.html
> <https://github.com/AustinCodingAcademy/intro-workbook/pull/15#discussion_r62687874>
> :
>
> >  <body>
> > +  <!-- Defines page header -->
> > +  <div>
> > +    <img src="C:\Users\salva\OneDrive\Documents\GitHub\intro-workbook\02LessonTwo\homework\artwork1.jpg"/>
>
> You don't necessarily need to list the path from C: to 02LessonTwo; if
> it's in the same repository as homework, you can change this to
> "./artwork1.jpg"
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/AustinCodingAcademy/intro-workbook/pull/15/files/291b680114647074ef137db59e4ed03958379413#r62687874>
>